mod/01_core/set/all/trash.rb

```diff
@@ -31,9 +31,9 @@ def delete!
     errors.add :delete, "#{name} is an indestructible rule"
   end
   
-  # if account && Card::Revision.find_by_creator_id( self.id )  #ask ethan
-  #   errors.add :delete, "Edits have been made with #{name}'s user account.\n  Deleting this card would mess up our revision records."
-  # end
+  if account && Card::Act.find_by_actor_id( self.id )
+    errors.add :delete, "Edits have been made with #{name}'s user account.\nDeleting this card would mess up our history."
+  end
```

Comment:
As long as the other branch gets merged first, this shouldn't be a problem, but if that changes, we better clean this up first.